src/org/opendatakit/briefcase/util/ExportToCsv.java

```diff
@@ -74,42 +74,37 @@
 
   private static final Log log = LogFactory.getLog(ExportToCsv.class);
 
-  File outputDir;
-  File outputMediaDir;
-  String baseFilename;
-  BriefcaseFormDefinition briefcaseLfd;
-  TerminationFuture terminationFuture;
-  Map<TreeElement, OutputStreamWriter> fileMap = new HashMap<TreeElement, OutputStreamWriter>();
-  Map<String, String> fileHashMap = new HashMap<String, String>();
-
-  boolean exportMedia = true;
-  Date startDate;
-  Date endDate;
-  boolean overwrite = false;
   int totalFilesSkipped = 0;
   int totalInstances = 0;
   int processedInstances = 0;
-
-
-  public ExportToCsv(File outputDir, BriefcaseFormDefinition lfd, TerminationFuture terminationFuture, Date start, Date end) {
-    this(outputDir, lfd, terminationFuture, lfd.getFormName(), true, false, start, end);
-  }
-
-  public ExportToCsv(File outputDir, BriefcaseFormDefinition lfd, TerminationFuture terminationFuture, String filename, boolean exportMedia, Boolean overwrite, Date start, Date end) {
+  private final Map<TreeElement, OutputStreamWriter> fileMap = new HashMap<TreeElement, OutputStreamWriter>();
+  private final Map<String, String> fileHashMap = new HashMap<String, String>();
+  private final TerminationFuture terminationFuture;
+  private final File outputDir;
+  private final BriefcaseFormDefinition briefcaseLfd;
+  private final String baseFilename;
+  private final boolean exportMedia; // true by default?
+  private final boolean overwrite; // false by default?
```

Comment:
I’m confused by these “by default?” comments.

Reply:
I forgot to remove those. They had those values by default prior to these changes and I commented before transforming that into a final variable...

Reply:
I sometimes flag such temporary changes with `// todo remove` or some such.